common/request_schema.go

```diff
@@ -43,6 +43,7 @@ var SchemaDataTypes = struct {
 	Platform       string
 	Architecture   string
 	SensorSelector string
+	Event          string
```

Comment:
Can we make that EventName? To make sure there is no confusion that it's a name vs actual event content somehow.